packages/editor-ui/src/api/workflowHistory.ts

```diff
@@ -0,0 +1,31 @@
+import type { IRestApiContext } from '@/Interface';
+import { makeRestApiRequest } from '@/utils';
+import type { WorkflowHistory, WorkflowVersion } from '@/types/workflowHistory';
+import type { HasAtLeastOneKey } from '@/utils/typeHelpers';
+import { valuesToString } from '@/utils/objectUtils';
+
+const workflowHistoryApiRoot = '/workflow-history';
+
+export const getWorkflowHistory = async (
```

Comment:
where are the tests?

Reply:
I'll add them later in a separate PR.

Reply:
maybe then we should not merge this to master?

Reply:
It will not be executed until we enable the feature

Reply:
I'll ask you for the review for the tests of this feature ;)

Reply:
I am sorry but as a reviewer, it will be impossible for me to go back and check the quality of coverage after the fact when you decide to add the PR with the tests. Or as often seems to happen, that tests are added at all.